scripts/check_missing_gitignore_for_template_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+ERROR=0
+
+for FILE in $(git ls-files | grep '\.sql\.in')
+do
+  DIRNAME=$(dirname "${FILE}")
+  FILENAME=$(basename "${FILE}" .sql.in)
+  GITIGNORE=${DIRNAME}/.gitignore
+  if [ -f "${GITIGNORE}" ]; then
+    if ! grep --silent "${FILENAME}-\*.sql" "${GITIGNORE}"; then
```

Comment:
Just for good measure to avoid problems with strange file names, even if unlikely

```suggestion
    if ! fgrep --silent -e "${FILENAME}-*.sql" "${GITIGNORE}"; then
```

Reply:
Fixed!